Dockerfile

```diff
@@ -16,9 +16,8 @@ RUN apt-get update -q \
     && rm -r /var/lib/apt/lists/*
 
 
-# ARG SWIFT_LINT_VER=0.39.1 # swiftlint
-# RUN git clone -b $SWIFT_LINT_VER --single-branch --depth 1 https://github.com/realm/SwiftLint.git _SwiftLint # swiftlint
-# RUN cd _SwiftLint && git submodule update --init --recursive; make install # swiftlint
+# ARG SWIFT_LINT_VER=0.40.3 # swiftlint
```

Comment:
I got an error here: `Error response from daemon: Dockerfile parse error line 19: ARG requires exactly one argument`

After removing `# swiftlint` it worked fine. Though this would require the `sed`-command to be changed.
No problem at the next line.

Reply:
Really interesting 🤔 
The reason why the swiftlint is needed is to tell the script which line uncomment, I will give it a look.
Thank you very much for checking it :)

Reply:
I've solved this by adding the version to the next line